svc/receiver.go

```diff
@@ -2,36 +2,38 @@ package svc
 
 import (
 	"fmt"
+
 	"github.com/hyprspace/hyprspace/config"
 	"github.com/libp2p/go-libp2p/core/network"
 )
 
 func (sn *ServiceNetwork) streamHandler() func(network.Stream) {
 	return func(stream network.Stream) {
 		if _, ok := config.FindPeer(sn.config.Peers, stream.Conn().RemotePeer()); !ok {
+			logger.Debug("Could not find peer")
```

Comment:
```suggestion
			logger.Debug("Connection attempt from untrusted peer")
```
This check is about making sure the peer is in the list of trusted peers. Untrusted peers are not allowed on this protocol.